packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -399,6 +399,15 @@ class EditableText extends StatefulWidget {
   /// Defaults to EdgeInserts.all(20.0).
   final EdgeInsets scrollPadding;
 
+  static bool get debugDeterministicCursor => _debugDeterministicCursor;
```

Comment:
Can you please add a doc comment explaining this setting?

Reply:
Will do.